data/json/monsters/robofac_robots.json

```
@@ -32,5 +32,30 @@
     "death_drops": {  },
     "death_function": { "corpse_type": "BROKEN" },
     "flags": [ "SEES", "NOHEAD", "ELECTRONIC", "COLDPROOF", "IMMOBILE", "NO_BREATHE", "STUN_IMMUNE" ]
+  },
+  {
+    "id": "mon_robofac_camspy",
+    "type": "MONSTER",
+    "copy-from": "mon_eyebot",
+    "name": { "str": "Hub 01 camspy", "str_pl": "Hub 01 camspies" },
+    "description": "A white and brown quadcopter drone mounting what looks to be a camera dome and a large antenna array.  It calmly hovers a comfortable distance above the ground, surveying its surroundings with cold indifference.",
+    "default_faction": "robofac_spy",
+    "tracking_distance": 10,
+    "aggression": 50,
+    "special_attacks": [ [ "DEPART", 34560 ] ],
+    "death_drops": { "groups": [ [ "robots", 4 ], [ "eyebot", 1 ] ] },
+    "death_function": { "corpse_type": "BROKEN" },
+    "flags": [
```

```suggestion
    "death_function": { "corpse_type": "BROKEN" },
    "path_settings": { "max_dist": 20, "avoid_traps": true },
    "flags": [
        "PATH_AVOID_DANGER_1",
```
Might need a touch of reformatting. Looks good!